exporter/sumologicexporter/sender.go

```diff
@@ -263,6 +269,8 @@ func (s *sender) sendMetrics(ctx context.Context, flds fields) ([]metricPair, er
 			formattedLine = s.prometheusFormatter.metric2String(record)
 		case Carbon2Format:
 			formattedLine = carbon2Metric2String(record)
+		case GraphiteFormat:
+			formattedLine = s.graphiteFormatter.metric2String(record)
```

Comment:
For now let's leave this is as but I'd suggest to rethink how this is written and for instance use a `formatter` interface, make `func carbon2Metric2String()` into a struct that also implements it (the same as graphite and prometheus ones) and then just use a field this `interface` type.

Then there would be no need to `switch` on `s.config.MetricFormat` but only call `formatter.format()` and be done with it. Maybe even extend it further so that each formatter (maybe called differently at that point) would know what header needs to be added and then just call `header()` on formatter (also avoiding need of switching on `s.config.MetricFormat`.